internal/sqlparser/testdata/valid-up/test03/01.golden.sql

```diff
@@ -11,6 +11,10 @@ BEGIN
                          v_url ALIAS FOR $3;
                      BEGIN ''; 
 
+-- 
```

Comment:
This fixes the issue reported here.

https://github.com/pressly/goose/issues/198#issuecomment-778108480

Once we're in the middle of a statement, we preserve comments.

Furthermore, we preserve whitespace which was reported here:

https://github.com/pressly/goose/issues/200#issue-556340796

And initially fixed here: https://github.com/pressly/goose/pull/372 (I've captured this test)